openshift/templates/kofax-error-cronjobs.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+items:
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: dps-cronjob
+  spec:
+    concurrencyPolicy: Replace
+    failedJobsHistoryLimit: 1
+    jobTemplate:
+      metadata:
+        creationTimestamp: null
+      spec:
+        template:
+          metadata:
+            creationTimestamp: null
+            labels:
+              name: dps-job
+              parent: dps-cronjob
+          spec:
+            containers:
+            - args:
+              - -c
+              - 'curl -XPOST ''http://dps-email-worker/dpsemailworker/job/error''
+                --header ''Content-Type: application/json'''
+              command:
+              - /bin/sh
+              image: centos:centos7
+              imagePullPolicy: Always
+              name: dps-job-pod
+              resources: {}
+              terminationMessagePath: /dev/termination-log
+              terminationMessagePolicy: File
+            dnsPolicy: ClusterFirst
+            restartPolicy: OnFailure
+            schedulerName: default-scheduler
+            securityContext: {}
+            terminationGracePeriodSeconds: 30
+    schedule: '*/30 * * * *'
+    startingDeadlineSeconds: 200
+    successfulJobsHistoryLimit: 3
+    suspend: true
+  status:
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: spd-cronjob
+  spec:
+    concurrencyPolicy: Replace
+    failedJobsHistoryLimit: 1
+    jobTemplate:
+      metadata:
+        creationTimestamp: null
+      spec:
+        template:
+          metadata:
+            creationTimestamp: null
+            labels:
+              name: spd-job
+              parent: spd-cronjob
+          spec:
+            containers:
+            - args:
+              - -c
+              - 'curl -XPOST ''http://spd-email-worker/dpsemailworker/job/error''
```

Comment:
are you duplicating this in openshift ?

Reply:
I can parameterize that. those are literally what is in OpenShift Dev/Test/Prod already